setup-kubectl/action.yml

```diff
@@ -5,6 +5,7 @@ inputs:
     description: 'Version of kubectl'
     required: true
     default: 'latest'
+    deprecationMessage: "This action is ARCHIVED and will not receive any updates, update your workflows to use the new action azure/setup-kubectl@v1"setup-kubectl

```

Comment:
This is not a valid yaml. Remove the `setup-kubectl` part or keep it inside the quotes

Reply:
Made the change as suggested. Plz chk.